tsconfig.json

```diff
@@ -27,6 +27,9 @@
     "paths": {
       "@state": [
         "src/app/state/index"
+      ],
+      "@tests": [
+        "src/app/tests/index"
```

Comment:
This also doesn't much sense to me. Is it because `@tests` is being imported from non test files?

Reply:
This is alias to file. In this case to index.ts file which exports all mocks/stubs etc. That way, i can remove paths like `../../../../../../something/.../file.ts`

Reply:
Ah yes, right, I think you might have explained this to me before 😆